lib/streamlit/elements/altair.py

```diff
@@ -34,6 +34,9 @@ def line_chart(dg, data=None, width=0, height=0, use_container_width=True):
         the chart's spec. As a result this is easier to use for many "just plot
         this" scenarios, while being less customizable.
 
+        In the case where this function does not guess the data specification
```

Comment:
Would slightly rewrite these to use active voice:

"If `st.line_chart` does not guess the data specification correctly, try specifying your desired chart using `st.altair_chart`."